nwb_conversion_tools/gui/classes/forms_basic.py

```diff
@@ -228,9 +247,11 @@ def fill_fields_info(self):
             # Required fields get a red star in their label
             if 'default' not in field:
                 required = True
+                default = 'required: '+ str(field['type'])
```

Comment:
?

Reply:
Fills in the form like so:
![image](https://user-images.githubusercontent.com/11567561/84208774-7eeaaa00-aa82-11ea-80a0-ffd16326a6b7.png)

Reply:
ok that's fine